### PR TITLE
Add Ocean's 8

### PR DESCRIPTION
**Pull request by igor-nm**
Ocean's 8 movie in json file

### CODE REVIEW

**Comment by igor-nm** [comment]
#73 Reference this issue

File: movies/2018/ocean-8.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "future": true,
+    "release-date": "2018-6-8",
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: Could you please change the format to 2018-06-08 😄

File: movies/2018/ocean-8.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "categories": [
+      "American Comedy"
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: Please change this to 'comedy', 'American Comedy' is not an accepted category 😄

File: movies/2018/ocean-8.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
...Having crossed paths with the Oceans before, Frazier immediately suspects Debbie, but her careful presence on the gala's video footage gives her an unbeatable alibi. She tells Frazier she may know who stole the necklace. Daphne visits Becker and sends Frazier a picture of the jewel Debbie planted. To further frame Becker, Debbie hires actresses posing as elderly socialites to sell the Toussaint pieces and deposit the money in Becker's name. Frazier takes Becker into custody. As the eight celebrate their victory, Lou reveals the heist's true target: while the gala was being evacuated, she and "The Amazing" Yen replaced a Met display of royal jewels with replicas, escaping with gems even more valuable than the Toussaint. With a larger-than-expected share of the score, each member of the team goes her separate way: Amita travels to Paris with a man she meets on Tinder, Weil pays off her debts and opens her own store, Constance buys a spacious loft in the city and becomes a YouTuber, Tammy expands her business in stolen goods, Nine Ball opens a pool bar, Daphne becomes a film director, Lou goes on a cross-country road trip, and Debbie visits Danny's grave with a martini in his honor.",
```
Comment (leonlafa) [inline movies/2018/ocean-8.json]: This summary is a little too long.   Could you please  use the following:

"Debbie Ocean gathers an all-female crew to attempt an impossible heist at New York City's yearly Met Gala."

**Comment by igor-nm** [comment]
Hi, I made a new PR because I deleted the fork that I had this PR. I'm a new contributor and I'm learning how is it work, sorry about my mistakes.
PR with requested updates: #145

**Comment by igor-nm** [comment]
These updates em correct PR is in #145 .
So if you can, close . this issue.

**Pull request closed without merging**